frontend/packages/helm-plugin/integration-tests/features/helm/install-helm-chart.feature

```diff
@@ -82,3 +82,54 @@ Feature: Install the Helm Release
              When user selects the Chart Version "0.2.0 / App Version 1.16.0 (Provided by Red Hat Helm Charts)"
              When user selects "Proceed" button from Change Chart version confirmation dialog
              Then user will see that the README is also updated with new chart version "0.2.0 / App Version 1.16.0 (Provided by Red Hat Helm Charts)"
+
+
+        @regression
+        Scenario: Certification filter in Helm Catalog Page
+            Given user is at Add page
+              And user has added multiple helm charts repositories with providerType annotations in index.yaml
+              And user has disabled the default Red Hat helm chart repo
```

Comment:
this step won't be needed anymore after https://github.com/openshift/console/pull/8983 goes in.

Reply:
Updated